web/help/GuideUnit1.md

```diff
@@ -355,9 +357,7 @@ favoriteColor = blue
 
     More on that later!
 
-Each of these lines is an **equation**, which says that two expressions are
-*equal*, or have the same value.  In math, you use equations in many
-ways, but in CodeWorld they are used specifically to *define variables*.
+Each of these is a type of **equation**, which says assign the value on the right to the name on the left. You have seen this type of equation in math, often with variables x and y. For example, in math, you might see x=10 and y=20.
```

Comment:
As above, we need to be very careful to give a mathematically correct definition of an equation.  Then, as a second step, explain which specific kinds of equations CodeWorld can understand.